ui/admin/tests/acceptance/authentication-test.js

```diff
@@ -124,10 +123,7 @@ module('Acceptance | authentication', function (hooks) {
 
   test('visiting auth method when the scope cannot be loaded is still allowed', async function (assert) {
     assert.expect(1);
-    setupOnerror(() => {
```

Comment:
Seems like this doesn't work anymore, possibly because ember changed what uses ember runloop in the most recent changes and this only catches things on the runloop.